netplan/cli/commands/apply.py

```diff
@@ -108,6 +108,10 @@ def command_apply(run_generate=True, sync=False, exit_on_error=True):  # pragma:
         # stop backends
         if restart_networkd:
             logging.debug('netplan generated networkd configuration changed, restarting networkd')
+            # Running 'systemctl daemon-reload' will re-run the netplan systemd generator,
+            # so let's make sure we only run it iff we're willing to run 'netplan generate'
```

Comment:
Typo, 'iff'.

Reply:
'iff' was used on purpose, with the meaning of 'if and only if'.